spec.bs

```diff
@@ -86,6 +86,9 @@ are:
 1. If [=this=]'s [=relevant global object=]'s [=associated Document=] is not [=allowed to use=] the
   "[=join-ad-interest-group=]" [=policy-controlled feature=], then [=exception/throw=] a
   "{{NotAllowedError}}" {{DOMException}}.
+1. Let |frameOrigin| be the [=relevant settings object=]'s [=environment settings object/origin=].
+1. If |frameOrigin| is an [=opaque origin=], then [=exception/throw=] a

```

Comment:
Looking into this a bit more, it actually does seem like this is not possible to ever hit this exception, since opaque origins aren't considered secure https://w3c.github.io/webappsec-secure-contexts/#potentially-trustworthy-origin so maybe let's just assert that this is not an opaque origin